x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/all/rules_table_filters/tags_filter_popover.tsx

```diff
@@ -41,9 +41,11 @@ export const TagsFilterPopoverComponent = ({
 }: TagsFilterPopoverProps) => {
   const [isTagPopoverOpen, setIsTagPopoverOpen] = useState(false);
   const [selectedTags, setSelectedTags] = useState<string[]>([]);
```

Comment:
```suggestion
  const [selectedTags, setSelectedTags] = useState<string[]>(tags ?? []);
```

Reply:
we dont want to select all the tags by default